docs/DeveloperGuide.adoc

```diff
@@ -909,6 +909,93 @@ Use case ends.
 +
 Use case ends.
 
+[discrete]
+=== Use case: UC3 - View a specific book
+
+*MSS*
+
+1.  User performs a search with some constraints (title, author, etc.).
+2.  Bibliotek lists books satisfying all the constraints.
```

Comment:
1. User obtains a list of books by listing (UC1) or searching (UC2).
2. User requests to view more information on a specific book in the list.

Reply:
also underline and bold references